litmus-portal/frontend/src/components/Dashboard/WorkflowTabs/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { AppBar, Box, Typography } from '@material-ui/core';
+import { createStyles, withStyles } from '@material-ui/core/styles';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import React from 'react';
+import { history } from '../../../redux/configureStore';
+import ButtonFilled from '../../Button/ButtonFilled';
+import Loader from '../../Loader';
+import BrowseWorkflow from '../BrowseWorkflow';
+import useStyles from './styles';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+const TabPanel: React.FC<TabPanelProps> = ({ children, index, value }) => {
+  return (
+    <div role="tabpanel" hidden={value !== index}>
+      {value === index && (
+        <Box p={3}>
+          <>{children}</>
+        </Box>
+      )}
+    </div>
+  );
+};
+interface StyledTabProps {
+  label: string;
+}
+const StyledTab = withStyles((theme) =>
+  createStyles({
+    root: {
+      textTransform: 'none',
+      color: 'rgba(0,0,0,0.5)',
```

Comment:
This too like the function we talked about...you can hit me up if you need help with this